### ISSUE TITLE
ImportError: cannot import name JIRAError

### ISSUE THREAD

**Issue by arulrajnet** [alisaifee/jira-cli#21]
Installed jira-cli with pip.

jira-cli 2.0.11
Python 2.7.6

```
jira-cli list projects
```

Error is 

```
Traceback (most recent call last):
  File "/usr/local/bin/jira-cli", line 9, in <module>
    load_entry_point('jira-cli==2.0.11', 'console_scripts', 'jira-cli')()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 351, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2363, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2088, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/usr/local/lib/python2.7/dist-packages/jira_cli-2.0.11-py2.7.egg/jiracli/interface.py", line 6, in <module>
    from jira.utils import JIRAError
ImportError: cannot import name JIRAError
```

**Comment by alisaifee** [comment]
Thanks @arulrajnet. Looks like [this](https://github.com/pycontribs/jira/commit/adf8e07619f97ed19cbd3e85271e411adfaaf1f4) happened recently. I'll make a patch and release it in a bit.

**Comment by alisaifee** [comment]
Fixed in `2.0.12` on pypi
